GOVERNANCE.md

```diff
@@ -72,8 +72,10 @@ membership beyond these rules.
 
 The CTC may add additional members to the CTC by a standard CTC motion.
 
-A CTC member may be removed from the CTC by voluntary resignation, or by
-a standard CTC motion.
+When a CTC member's participation in CTC matters has become minimal for a
```

Comment:
> There is no specific set of requirements or qualifications for CTC membership beyond these rules

This is still correct, but if CTC members don't participate in meetings there is no point in being on the CTC.